src/models/matches.rs

```diff
@@ -201,13 +203,16 @@ impl Match {
           self.associated_comma = Some(sibling.range().into());
           current_node = sibling;
           continue; // Continue the inner loop (i.e. evaluate next sibling)
-        } else if delete_comments
-          && self._is_comment_safe_to_delete(&sibling, node, piranha_arguments, trailing)
-        {
-          // Add the comment to the associated matches
-          self.associated_comments.push(sibling.range().into());
-          current_node = sibling;
-          continue; // Continue the inner loop (i.e. evaluate next sibling)
+        } else if self._is_comment_safe_to_delete(&sibling, node, piranha_arguments, trailing) {
+          if !keep_comment_regexes.iter().any(|pattern| {
+            Regex::new(pattern)
+              .map(|re| re.is_match(sibling.utf8_text(code.as_bytes()).unwrap()))
+              .unwrap_or(false)
```

Comment:
I'm just wondering if we should pre-compile the regexes instead of doing it every time. But I don't know rust enough to say whether that's worth it :)